### PR TITLE
Cext 2835

### PR DESCRIPTION
**Pull request by AjazSumaiya**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adobe/plugin-hooks",
-  "version": "0.1.1",
+  "version": "0.2.0-beta1",
```
Comment (dthampy) [inline package.json:3]: @AjazSumaiya are we publishing this as beta?

Comment (AjazSumaiya) [reply package.json:3]: to test i was thinking of publishing beta first

Comment (amolina-adobe) [reply package.json:3]: To publish as a legitimate beta, we'll need to add the beta tag in the [.github/workflows
/publish-to-npm.yml](https://github.com/adobe/plugin-hooks/blob/develop/.github/workflows/publish-to-npm.yml#L22-L24) file (and later revert it).

**Review by amolina-adobe — DISMISSED** [review]

**Review by simrangulati — DISMISSED** [review]

**Review by simrangulati — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (AjazSumaiya) [reply package.json:3]: updated the tag

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adobe/plugin-hooks",
-  "version": "0.1.1",
+  "version": "0.2.0",
```
Comment (amolina-adobe) [inline package.json:3]: Please leave this suffixed this with` -beta` based on our nomenclature.

**Review by dani97 — APPROVED** [review]

**Review by amolina-adobe — APPROVED** [review]

**Pull request merged**